src/RealtimeEffectPanel.cpp

```diff
@@ -1006,7 +1006,7 @@ class RealtimeEffectListWindow : public wxScrolledWindow
       mAddEffectHint = addEffectHint;
 
       //TODO: set link
-      auto addEffectTutorialLink = safenew ThemedWindowWrapper<wxHyperlinkCtrl>(this, wxID_ANY, _("Watch video"), "https://www.audacityteam.org", wxDefaultPosition, wxDefaultSize, wxHL_ALIGN_LEFT | wxHL_CONTEXTMENU);
+      auto addEffectTutorialLink = safenew ThemedWindowWrapper<wxHyperlinkCtrl>(this, wxID_ANY, _("Watch video"), "https://www.audacityteam.org/3.2.0-video", wxDefaultPosition, wxDefaultSize, wxHL_ALIGN_LEFT | wxHL_CONTEXTMENU);
```

Comment:
Hmm, probably we should have a different link (probably linking to the same page for now).

This will allow us to add RT specific tutorial, if we choose to do so.

Reply:
that's a good idea